doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -155,3 +158,40 @@ You can query the ``refcats`` collection to see whether your htm shards appear:
     butler query-datasets --collections refcats REPO
 
 For LSST staff using ``lsst-devl``, see the `Reference catalogs policy <https://developer.lsst.io/services/datasets.html#reference-catalogs>`_ in the Developer Guide for additional policy about adding reference catalogs to the common repo.
+
+.. _lsst.meas.algorithms-refcat-existing:
+
+5. What to do with existing reference catalogs?
+===============================================
+
+Already existing reference catalogs (for example, the PS1 or Gaia DR2 catalogs that were used in gen2 butlers) can also be ingested into the butler, negating the need to convert them to the LSST format yourself.
+To ingest an already existing converted reference catalog, first create a suitable filename to htm7-index astropy lookup table, and then follow the steps above to `ingest the files into the butler <lsst.meas.algorithms-refcat-ingest>`.
+
+This is an example script that constructs a conversion lookup table for all converted FITS files in the `refcat_dir` directory, outputting to the `out_dir` directory:
+
+.. code-block:: python
+
+    import os
+    import glob
+    import astropy.table
+
+    refcat_dir = "/path/to/my/refcat/directory"
```

Comment:
In my comment above, I suggested using `/datasets/refcats/htm/v1/gaia_dr2_20200414` here, to make it clear what exactly the this path should look like for an existing refcat. The rest of this example refers to lsst-devl (e.g. in the GaiaSource path, and the number of cores to use in the convert), so I don't think it hurts to do the same here.